test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -180,10 +180,26 @@ class test_ucp_sockaddr : public ucp_test {
     void start_listener(ucp_test_base::entity::listen_cb_type_t cb_type,
                         const struct sockaddr* addr, size_t addrlen)
     {
-        ucs_status_t status = receiver().listen(cb_type, addr, addrlen,
-                                                get_ep_params());
+        ucs_status_t status;
+
+        do {
+            status = receiver().listen(cb_type, addr, addrlen, get_ep_params());
+        } while (status == UCS_ERR_BUSY);
+
         if (status == UCS_ERR_UNREACHABLE) {
-            UCS_TEST_SKIP_R("cannot listen to " + ucs::sockaddr_to_str(addr));
+            UCS_TEST_SKIP_R("cannot listen to " + m_test_addr.to_str());
```

Comment:
start_listener accepts 'addr' but also uses 'm_test_addr'. pls use only one of them